src/angular-local-storage.js

```diff
@@ -302,7 +302,10 @@ angularLocalStorage.provider('localStorageService', function() {
           expiryDate.setTime(expiryDate.getTime() + (-1 * 24 * 60 * 60 * 1000));
           expiry = "; expires=" + expiryDate.toGMTString();
           value = '';
-        } else if (isNumber(daysToExpiry) && daysToExpiry !== 0) {
+          //if the value is undefined set the cookie to session cookie
+        } else if (daysToExpiry == undefined) {
```

Comment:
What about passing the expiry as 0? Check here #246. I would change this `else if` to something like this:
```
else if (typeof daysToExpiry !== 'undefined') {
  if (isNumber(daysToExpiry) && daysToExpiry !== 0) {
    // set the expiration 
  } // otherwise without the expires it will be a session cookie
}
```